index.html

```diff
@@ -14,46 +14,258 @@
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <h4 class="connect">LET’S CONNECT</h4>
+
+        <ul class="social-icons">
+          <li><span class="iconify" data-icon="iconoir:github"></span></li>
+          <li><span class="iconify" data-icon="eva:linkedin-fill"></span></li>
+          <li><span class="iconify" data-icon="fontisto:angelist"></span></li>
+          <li><span class="iconify" data-icon="ei:sc-twitter"></span></li>
+          <li><span class="iconify" data-icon="bx:bxl-medium"></span></li>
+        </ul>
+
+        <button class="resume">Get my resume</button>
+
+        <ul class="language-list">
+          <li class="langguage">
+            <h5 class="icondown-language">Langguages</h5>
+            <div class="icon-down"></div>
+          </li>
+
+          <li class="major-lang">
+            <img src="img/Ellipse 1.png" class="eclipse-language" />
+            <p class="technology-language">JavaScript</p>
+          </li>
+          <li class="major-lang">
+            <img src="img/Ellipse 1 (1).png" class="eclipse-language" />
+            <p class="technology-language">HTML</p>
+          </li>
+          <li class="major-lang">
+            <img src="img/Ellipse 1 (2).png" class="eclipse-language" />
+            <p class="technology-language">CSS</p>
+          </li>
+          <li class="major-lang">
+            <h5 class="about-h5">Frameworks</h5>
+            <div class="icon-right"></div>
+          </li>
+
+          <li class="major-lang">
+            <h5 class="about-h5">Skills</h5>
+            <div class="icon-right"></div>
+          </li>
+        </ul>
+      </section>
+
+      <section class="contact-section">
+        <div class="ellipse-rectangle">
+          <img src="img/Ellipse 12.png" alt="Eclipse-image" class="Ellipse12" />
+          <div class="ellipse-rectangle2">
+            <img
+              src="img/Rectangle 4.png"
+              alt="Rectangle-image"
+              class="rectangle4"
+            />
+            <img src="img/Ellipse 3.png" alt="Eclipse-image" class="Ellipse3" />
+          </div>
+        </div>
+        <div class="vector"></div>
+        <h6 class="contact-me">Contact me</h6>
+        <p class="apply">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+
+        <form action="form">
+          <input
+            type="text"
+            placeholder="Ismail"
+            class="contactsection-input"
+          />
+          <input
+            type="text"
+            placeholder="adismail4@yahoo.com"
+            class="contactsection-input"
+          />
+          <textarea>Write your message here.</textarea>
+
+          <div class="button-eclipse">
+            <button class="contactsection-button">Get It Touch</button>
```

Comment:
- [ ] Please make sure your design matches the original template :pray: 

## Screen
![image](https://user-images.githubusercontent.com/28903404/143721301-19d1112a-1979-4201-b26f-7c2637aa72cd.png)